db/models/index.js

```diff
@@ -19,9 +19,10 @@ class SequelizeFactory {
             define: this.config.define,
             storage: this.config.storage,
             transactionType: this.config.transactionType,
-            retry: {
-                max: this.config.retry.max
-            },
+            // TODO: docker e2e fails bc this is missing in config. is it needed?
```

Comment:
Please remove the comment and make sure the app still works.

Reply:
It was not in e2e. In a normal or dev it worked fine, but I will also re-check

Reply:
Also nothing changed. With this config docker e2e fails. Leaving it as is for now.